### PR TITLE
Use netlink collectd plugin instead of interface

### PR DESCRIPTION
**Pull request by elemoine**
This patch replaces the "interface" collectd plugin by the "netlink" one. The "netlink" plugin provides the same metrics as "interface" but plus other metrics such as the number of dropped packets.

### CODE REVIEW

**Comment by elemoine** [comment]
Note for developers:

For the `netlink.py` module to be added the following is necessary:

```
cd /usr/share/salt-formulas/env/_formulas/linux
cp /usr/share/salt-formulas/env/_formulas/linux/_modules/* /usr/share/salt-formulas/env/_modules/
salt '*' saltutil.sync_modules
```

File: linux/meta/collectd.yml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 local_plugin:
-  linux_network_interface:
-    plugin: interface
+  linux_netlink:
```
Comment (simonpasquier) [inline linux/meta/collectd.yml]: s/linux_netlink/linux_network_netlink/ to be consistent with the rest

Comment (elemoine) [inline linux/meta/collectd.yml]: Indeed, will change…

Comment (elemoine) [inline linux/meta/collectd.yml]: Done.

**Pull request merged**